docs/_docs/memory-configuration/data-regions.adoc

```diff
@@ -84,51 +84,103 @@ tab:C++[unsupported]
-====
-
-Presently, Ignite warm-up strategy implies loading data into the region, starting with indexes, until it runs out of free space.
+Presently, the Ignite warm-up strategy implies loading data into all or specific data regions of Ignite, starting with indexes, until it runs out of free space. It can be configured both for all regions (by default) or for each region separately.
 
 To warm up all data regions, pass the configuration parameter `LoadAllWarmUpStrategy` to the `DataStorageConfiguration#setDefaultWarmUpConfiguration` as follows:
 
-[source, java]
+:xmlFile: code-snippets/xml/warm-up-strategy.xml
+:javaFile: {javaCodeDir}/WarmUpStrategy.java
+
+[tabs]
+--
+tab:XML[]
+[source,xml]
 ----
-setDefaultWarmUpConfiguration(loadAllWarmUpStrategy)
+include::{xmlFile}[tag=warm-all-regions,indent=0]
 ----
+tab:Java[]
+[source,java]
+----
+include::{javaFile}[tag=warm-all-regions,indent=0]
+----
+tab:C#/.NET[unsupported]
+
+tab:C++[unsupported]
+--
+
 
 To warm up a specific data region, pass the configuration parameter `LoadAllWarmUpStrategy` to the `DataStorageConfiguration#setWarmUpConfiguration` as follows:
 
-[source, java]
+
+[tabs]
+--
+tab:XML[]
+[source,xml]
 ----
-setWarmUpConfiguration(loadAllWarmUpStrategy)
+include::{xmlFile}[tag=warm-specific-region,indent=0]
 ----
+tab:Java[]
+[source,java]
+----
+include::{javaFile}[tag=warm-specific-region,indent=0]
+----
+tab:C#/.NET[unsupported]
 
-To stop warming up all data regions, pass the configuration parameter `NoOpWarmUpStrategy` to the `DataStorageConfiguration#setDefaultWarmUpConfiguration` as follows:
+tab:C++[unsupported]
+--
 
-[source, java]
+
+To disable the warm-up for all data regions, pass the configuration parameter `NoOpWarmUpConfiguration` to the `DataStorageConfiguration#setDefaultWarmUpConfiguration` as follows:
+
+
+[tabs]
+--
+tab:XML[]
+[source,xml]
+----
+include::{xmlFile}[tag=disable-all-regions,indent=0]
 ----
-setDefaultWarmUpConfiguration(noOpWarmUpStrategy)
+tab:Java[]
+[source,java]
 ----
+include::{javaFile}[tag=disable-all-regions,indent=0]
+----
+tab:C#/.NET[unsupported]
 
-To stop warming up a specific data region, pass the configuration parameter `NoOpWarmUpStrategy` to the `DataStorageConfiguration#setWarmUpConfiguration` as follows:
+tab:C++[unsupported]
+--
 
-[source, java]
+
+To disable the warm-up for a specific data region, pass the configuration parameter `NoOpWarmUpStrategy` to the `DataStorageConfiguration#setWarmUpConfiguration` as follows:
+
+[tabs]
+--
+tab:XML[]
+[source,xml]
+----
+include::{xmlFile}[tag=disable-specific-region,indent=0]
+----
+tab:Java[]
+[source,java]
 ----
-setWarmUpConfiguration(noOpWarmUpStrategy)
+include::{javaFile}[tag=disable-specific-region,indent=0]
 ----
+tab:C#/.NET[unsupported]
 
-You can also stop the cache warming up process by using `control.sh` and JMX.
+tab:C++[unsupported]
+--
+
+
+You can also disable the cache warm-up by using `control.sh` and JMX.
```

Comment:
We can stop it if the user needs it, for example it takes too long.

Reply:
Fixed.